packages/okta-auth-js/lib/browser/browser.js

```diff
@@ -219,105 +218,104 @@ proto.signIn = function (opts) {
   });
 };
 
-// Ends the current application session, clearing all local tokens
-// Optionally revokes the access token
-// Ends the user's Okta session using the API or redirect method
-proto.signOut = function (options) {
+// Ends the current application session without redirecting to Okta. SSO session will remain active.
+proto.closeSession = function closeSession() {
+  var sdk = this;
+  
+  // Clear all local tokens
+  sdk.tokenManager.clear();
+
+  return sdk.session.close() // DELETE /api/v1/sessions/me
+  .catch(function(e) {
+    if (e.name === 'AuthApiError') {
+      // Most likely cause is session does not exist or has already been closed
+      // Could also be a network error. Nothing we can do here.
```

Comment:
I am changing this to only capture the specific error related to session not found. All other errors will throw